ci/kubetest/test_redis_external.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import pytest
+
+from utils import (
+    cleanup_k8s,
+    create_namespace_if_not_exists,
+    helm_install,
+    install_custom_resources,
+    is_posthog_healthy,
+    wait_for_pods_to_be_ready,
+)
+
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger()
+
+HELM_INSTALL_CMD = """
+helm upgrade \
+    --install \
+    --timeout 30m \
+    -f ../../ci/values/kubetest/test_redis_external.yaml \
+    --create-namespace \
+    --namespace posthog \
+    posthog ../../charts/posthog \
+    --wait-for-jobs \
+    --wait
+"""
+
+
+@pytest.fixture
+def setup(kube):
+    cleanup_k8s()
+    create_namespace_if_not_exists()
+    install_custom_resources("./custom_k8s_resources/redis_external.yaml")
+    helm_install(HELM_INSTALL_CMD)
+    wait_for_pods_to_be_ready(kube)
+
+
+def test_helm_install(setup, kube):
```

Comment:
non-blocking

Not in love with making tests dependent. Maybe we could go with making the setup fixture scope="module" instead?

Reply:
https://github.com/PostHog/charts-clickhouse/issues/265